integrationtest/backend-tck/src/main/java/org/hibernate/search/v6poc/integrationtest/backend/tck/MultiTenancyIT.java

```diff
@@ -6,8 +6,524 @@
  */
 package org.hibernate.search.v6poc.integrationtest.backend.tck;
 
+import static org.hibernate.search.v6poc.util.impl.integrationtest.common.assertion.DocumentReferencesSearchResultAssert.assertThat;
+import static org.hibernate.search.v6poc.util.impl.integrationtest.common.assertion.ProjectionsSearchResultAssert.assertThat;
+import static org.hibernate.search.v6poc.util.impl.integrationtest.common.stub.mapper.StubMapperUtils.referenceProvider;
+
+import java.util.List;
+
+import org.hibernate.search.v6poc.backend.document.DocumentElement;
+import org.hibernate.search.v6poc.backend.document.IndexFieldAccessor;
+import org.hibernate.search.v6poc.backend.document.IndexObjectFieldAccessor;
+import org.hibernate.search.v6poc.backend.document.model.IndexSchemaElement;
+import org.hibernate.search.v6poc.backend.document.model.ObjectFieldStorage;
+import org.hibernate.search.v6poc.backend.document.model.Store;
+import org.hibernate.search.v6poc.backend.document.model.spi.IndexSchemaObjectField;
+import org.hibernate.search.v6poc.backend.index.spi.ChangesetIndexWorker;
+import org.hibernate.search.v6poc.backend.index.spi.IndexManager;
+import org.hibernate.search.v6poc.backend.index.spi.IndexSearchTarget;
+import org.hibernate.search.v6poc.engine.spi.SessionContext;
+import org.hibernate.search.v6poc.integrationtest.backend.tck.util.rule.SearchSetupHelper;
+import org.hibernate.search.v6poc.search.DocumentReference;
+import org.hibernate.search.v6poc.search.SearchQuery;
+import org.hibernate.search.v6poc.util.SearchException;
+import org.hibernate.search.v6poc.util.impl.integrationtest.common.stub.StubSessionContext;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
 public class MultiTenancyIT {
```

Comment:
Well, I haven't added a test because the test only makes sense if the code stays exactly how it is. I mean, if we change the separator between the tenant and the id for instance, the test will be OK but the code might be wrong if we haven't updated the `replaceAll()` calls.

So, all in all, I think the test will mostly be useless.